modules/parquet/src/lib/parse-parquet-wasm.ts

```diff
@@ -1,11 +1,13 @@
 // eslint-disable
-import {readParquet} from 'parquet-wasm';
 import type {RecordBatch} from 'apache-arrow';
 import {Table, RecordBatchStreamReader} from 'apache-arrow';
+import {loadWasm} from './load-wasm/load-wasm-node';
 
 export async function parseParquet(arrayBuffer: ArrayBuffer, options): Promise<Table> {
+  const wasm = await loadWasm();
```

Comment:
Is this an expensive operation? Does it make sense to cache the result?

Reply:
Good question. In Node, the wasm can be `require`'d directly (synchronously), so all the function does is return the import: https://github.com/visgl/loaders.gl/blob/a745c404044d9dac3b9fd095c70b3a3fd78c2a09/modules/parquet/src/lib/wasm/load-wasm/load-wasm-node.ts#L1-L5

Outside Node, it fetches the wasm asynchronously, and the result is already cached inside of `loadWasm`:
https://github.com/visgl/loaders.gl/blob/a745c404044d9dac3b9fd095c70b3a3fd78c2a09/modules/parquet/src/lib/wasm/load-wasm/load-wasm-browser.ts#L4-L11

I figured it was cleaner to cache there rather than at the callsite of `loadWasm`.